StatBot/Cards/ActionButton.cs

```diff
@@ -4,91 +4,146 @@
+                            {
+                                stat.Subscribers.Add(userId);
+                                bd.Update(stat);
+                                activity.Text = $"Вы подписались на статистикиу №{buttonSettings.Statistic_Id}";
+                            }
+                            else
+                                activity.Text = "Вы уже подписаны на эту статистикиу";
+                            break;
                         }
-                        break;
-                    }
-                case "Unsubscribe":
-                    {
-                        string userId = activity.From.Id;
-                        if (stat.Subscribers.Contains(userId))
+
+                    case ((int)Constants.ActionTypes.Unsubscribe):
                         {
-                            stat.Subscribers.Remove(userId);
-                            bd.Update(stat);
-                            activity.Text = $"Вы отписались от статистики №{Id}";
+                            string userId = activity.From.Id;
+                            if (stat.Subscribers.Contains(userId))
+                            {
+                                stat.Subscribers.Remove(userId);
+                                bd.Update(stat);
+                                activity.Text = $"Вы отписались от статистики №{buttonSettings.Statistic_Id}";
+                            }
+                            else
+                                activity.Text = "Вы уже отписаны от этой статистики";
+                            break;
                         }
-                        else
+                    case ((int)Constants.ActionTypes.Back):
                         {
-                            activity.Text = "Вы уже отписаны от этой статистики";
+                            TuneStat tuneStat = new TuneStat();
+                            activity.Text = "";
+                            tuneStat.Run(activity);
+                            break;
                         }
-                        break;
-                    }
+                }
+
+            }
+            else
+            {
+                activity.Attachments = new List<Attachment>();
+                Attachment attachment;
+                var actions = new List<CardAction>();
+                EchoBot.ShowedButtons.Clear();
+                if (buttonSettings.ActionType == (int)Constants.ActionTypes.ShowTurn)
+                {
+                    actions.Add(new ActionButton(buttonSettings.Statistic_Id, Constants.TurnOn, (int)Constants.ActionTypes.TurnOn, Constants.NotShowButtons).Action);
+                    actions.Add(new ActionButton(buttonSettings.Statistic_Id, Constants.TurnOff, (int)Constants.ActionTypes.TurnOff, Constants.NotShowButtons).Action);
+                    //actions.Add(new ActionButton(buttonSettings.Statistic_Id, Constants.Back, (int)Constants.ActionTypes.Back, Constants.NotShowButtons).Action);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS125)